File: 30 days of code JAVA/day5.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.regex.*;

public class Solution {



    private static final Scanner scanner = new Scanner(System.in);

    public static void main(String[] args) {
        int n = scanner.nextInt();
        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
        
        int i;
        
        for(i=1;i<=10;i++)
        {
            System.out.println(n+" x "+i+" = "+ n*i);
        }
        
        

        scanner.close();
    }
}

File: 30daysOFcode_Scala/day5.scala
import java.io._
import java.math._
import java.security._
import java.text._
import java.util._
import java.util.concurrent._
import java.util.function._
import java.util.regex._
import java.util.stream._

object Solution {



    def main(args: Array[String]) {
        val stdin = scala.io.StdIn

        val n = stdin.readLine.trim.toInt
        for(i <- 1 to 10 ){
        println(n+" x "+i+" = "+ n*i)
        }
    }
}
